nucypher/blockchain/eth/sol/source/aragon/interfaces/IERC900History.sol

```diff
@@ -0,0 +1,8 @@
+pragma solidity ^0.6.5;
+
+// Minimum interface to interact with Aragon's Aggregator
+interface IERC900History {
```

Comment:
So this is just a partial implementation - or are we fully supporting ERC900?  How much of the standard is covered by existing staking methods?

Reply:
It's only a partial implementation. We could potentially do a full implementation as it only requires some minor changes in function signatures, but I don't think it's worth it. To date, this EIP has been only useful for view-only methods, which are the ones we implement here.